src/diffpy/utils/tests/test_tools.py

```diff
@@ -91,7 +91,6 @@ def test_get_user_info_with_home_conf_file(monkeypatch, inputs, expected, user_f
 
 @pytest.mark.parametrize("inputs, expected", params_user_info_with_local_conf_file)
 def test_get_user_info_with_local_conf_file(monkeypatch, inputs, expected, user_filesystem):
-    home_dir = _setup_dirs(monkeypatch, user_filesystem)
```

Comment:
Similarly, removed this because testing is done.

Reply:
don't remove